source/Mlos.Python/mlos/OptimizerEvaluationTools/unit_tests/TestEnvelopedWaves.py

```diff
@@ -39,15 +39,15 @@ def test_enveloped_waves(self):
             objectives_df = objective_function.evaluate_dataframe(random_params_df)
             assert ((objectives_df['y'] <= (num_params + vertical_shift)) & (objectives_df['y'] >= -num_params + vertical_shift)).all()
 
-    def test_random_configs(self):
-        for _ in range(100):
-            function_config = enveloped_waves_config_space.random()
-            objective_function = EnvelopedWaves(function_config)
-            random_params_df = objective_function.parameter_space.random_dataframe(100)
-            objectives_df = objective_function.evaluate_dataframe(random_params_df)
-            assert objective_function.output_space.get_valid_rows_index(objectives_df).equals(objectives_df.index)
+    @pytest.mark.parametrize('i', [i for i in range(10)])

```

Comment:
This took a long time, but it's just a synthetic function test that has not failed since November. Reducing num iterations.